hwy/ops/arm_neon-inl.h

```diff
@@ -4537,6 +4567,16 @@ HWY_API Vec128<uint64_t> Broadcast(Vec128<uint64_t> v) {
 
 // Signed
 template <int kLane>
+HWY_API Vec128<int8_t> Broadcast(Vec128<int8_t> v) {
+  static_assert(0 <= kLane && kLane < 16, "Invalid lane");
+  return Vec128<int8_t>(vdupq_n_s8(vgetq_lane_s8(v.raw, kLane)));
```

Comment:
I think we can skip the vgetq by using vdupq_laneq. Would you mind also updating the other instances of this?

Reply:
The issue is that ARMv7 NEON does not have the vdupq_laneq intrinsics that are available on AArch64.

Need to keep the `vdupq_n_s8(vgetq_lane_s8(v.raw, kLane))` for ARMv7.

Reply:
Oh, I see - the `#if` and especially the "No vdupq_laneq" comment were not visible in the diff, sorry about that. Never mind :)